docs/static/breaking-changes.asciidoc

```diff
@@ -34,20 +34,45 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
 * Configurations provided with `-f` or `config.path` will not be appended with `stdin` input and `stdout` output automatically.
 
 [float]
+=== Plugin Changes
 
+[float]
 ==== Elasticsearch output changes
 
-* The default `document_type` has changed from `logs` to `doc`.
+* The default `document_type` has changed from `logs` to `doc` for consistency with Beats.
   Furthermore, users are advised that Elasticsearch 6.0 deprecates doctypes, and 7.0 will remove them. 
   See https://www.elastic.co/guide/en/elasticsearch/reference/master/removal-of-types.html[Removal of Mapping Types] for more info.
 * The options `flush_size` and `idle_flush_time` are now obsolete.
 * Please note that the https://www.elastic.co/guide/en/elasticsearch/reference/6.0/mapping-all-field.html[_all] field is deprecated in 6.0.
  The new mapping template has been updated to reflect that. If you are using a custom mapping template you may need to update it to reflect that.
 
 [float]
+==== Kafka input changes
+
+* Upgrade Kafka client to version 0.11.0.0. Only supports brokers 0.10.1.x or later.
```

Comment:
I'd put this in the past tense: "Upgraded the Kafka client..."

Reply:
Should we add a bit more context here?

Upgraded Kafka client support to v0.11.0.0 which only supports Kafka brokers v0.10.1.x or later. For usage with Kafka versions older than v0.10.1.x, please consult the https://www.elastic.co/guide/en/logstash/current/plugins-inputs-kafka.html#_description_23[compatibility matrix] and install the correct plugin version.

Reply:
Sorry, missed the next line... looks fine.

Reply:
@acchen97 FYI - There are outstanding issues to update the compatibility matrices for the Kafka plugins - https://github.com/logstash-plugins/logstash-input-kafka/issues/214 and https://github.com/logstash-plugins/logstash-output-kafka/issues/142